Lib/test/test_audit.py

```diff
@@ -322,6 +331,16 @@ def test_assert_unicode(self):
         if returncode:
             self.fail(stderr)
 
+    @unittest.skipIf(not sys.is_remote_debug_enabled(), "Remote debugging is not enabled")
+    @unittest.skipIf(sys.platform != "darwin" and sys.platform != "linux" and sys.platform != "win32",
+                        "Test only runs on Linux, Windows and MacOS")
+    @unittest.skipIf(sys.platform == "linux" and not _supports_remote_attaching(),
+                        "Test only runs on Linux with process_vm_readv support")
+    @support.cpython_only
```

Comment:
Would it be possible to add a decorator to test.support to factorize these 4 decorators?